kdwsdl2cpp/CMakeLists.txt

```diff
@@ -59,16 +59,19 @@ set(SOURCES
   src/converter_simpletype.cpp
   src/creator.cpp
   src/elementargumentserializer.cpp
-  src/main.cpp
   src/namemapper.cpp
   src/settings.cpp
   src/typemap.cpp
 )
 
-add_executable(kdwsdl2cpp ${SOURCES})
-target_link_libraries(kdwsdl2cpp ${QT_LIBRARIES})
+set(SOURCES_EXE src/main.cpp)
+
+add_library(kdwsdl2cpp_lib STATIC ${SOURCES_LIB})
+add_executable(kdwsdl2cpp ${SOURCES_EXE})
+
+target_link_libraries(kdwsdl2cpp kdwsdl2cpp_lib ${QT_LIBRARIES})
 
 install(TARGETS kdwsdl2cpp EXPORT "KDSoapTargets"
-  RUNTIME DESTINATION ${INSTALL_RUNTIME_DIR}
```

Comment:
why did you remove RUNTIME?

Reply:
These are changes from another branch. They are not needed here.